README.md

```diff
@@ -1,3 +1,7 @@
+![Nuget](https://img.shields.io/nuget/v/morelinq?style=flat-square)
```

Comment:
The nuget badge should link to the nuget page for MoreLinq

Reply:
good catch! done in #813 because git said that this pull request's branch is protected I had to create a new one